test/shared-class.test.js

```diff
@@ -198,6 +198,25 @@ describe('SharedClass', function() {
     });
   });
 
+  describe('sharedClass.findMethodByName()', function() {
+    var sc;
+    var sm;
```

Comment:
What's the point of declaring the variables here, when they are always reinitialized in each test? Please keep them test-local.